app/services/users_service.rb

```diff
@@ -29,7 +29,10 @@ def register(email, password, organization_name)
 
     ActiveRecord::Base.transaction do
       result.user = User.create!(email:, password:)
-      result.organization = Organization.create!(name: organization_name, document_numbering: 'per_organization')
+
+      result.organization = Organizations::CreateService
+        .call(name: organization_name, document_numbering: 'per_organization')
+        .organization
```

Comment:
To raise the new service error (just in case):

```suggestion
      result.organization = Organizations::CreateService
        .call(name: organization_name, document_numbering: 'per_organization')
        .raise_if_error!
        .organization
```